repos/system_upgrade/el7toel8/actors/vimcheck/actor.py

```diff
@@ -0,0 +1,82 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_generic
+from leapp.models import InstalledRedHatSignedRPM, VimModel, Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+vimconfigs = {
+              'vim-minimal': '/etc/virc',
+              'vim-enhanced' : '/etc/vimrc'
```

Comment:
In this case, whatespaces is not expected before the colon
```suggestion
              'vim-enhanced': '/etc/vimrc'
```

Reply:
unresolved